packages/code-du-travail-api/src/server/routes/api.js

```diff
@@ -153,5 +155,34 @@ router.get(`${BASE_URL}/idcc`, async ctx => {
   });
 });
 
+/**
+ * Return annaire items matching the given departement.
+ *
+ * @example
+ * http://localhost:1337/api/v1/items/:id
+ *
+ * @param {string} :id The item ID to fetch.
+ * @returns {Object} Result.
+ */
+router.get(`${BASE_URL}/annuaire/search`, async ctx => {
+  let lon, lat;
+  let coordData = ctx.request.query.coord;
+  if (coordData) {
+    [lon, lat] = ctx.request.query.coord.split(":");
+  } else {
+    const address = ctx.request.query.q;
+    const response = await fetch(
+      `https://api-adresse.data.gouv.fr/search/?q=${address}&type=housenumber&limit=1`
```

Comment:
`https://api-adresse.data.gouv.fr` is a magic string that should come from a Magic contant file :)
[Can be done in another PR]